lib/teiserver_web/controllers/admin/user_controller.ex

```diff
@@ -1104,4 +1152,24 @@ defmodule TeiserverWeb.Admin.UserController do
         |> redirect(to: ~p"/teiserver/admin/user")
     end
   end
+
+
+  @spec delete_user(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def delete_user(conn, %{"id" => id}) do
```

Comment:
Similar comment regarding authentication.

Also, if the given parameters don't have an `id` key, the pattern match will fail and you're going to get a server error.
It would be better to add another catch all clause after this one to return back a 400 with a pre-rendered form (optional but nice to have)